### ISSUE TITLE
Directory and File naming/structure

### ISSUE THREAD

**Issue by mwhite192** [mwhite192/groupomania#1]
@AccessiT3ch - Hey Conor!  I finally have come code laid down! Yay!  I have just a few questions before I keep going to make sure I am following the best practices.  I know we covered some of these before, but I just want to be clear since I am actually building the site.  I will separate the questions so they are more organized as some relate to code while others deal with the more 'admin' side.  

So, the first question is am I following the correct file structure and naming conventions for each file and directory?  I am trying to remember what has to be capitalized and what doesn't.  If I remember correctly, component names should be capitalized.  I am just not sure about the css files or the component directory name.

**Comment by AccessiT3ch** [comment]
@mwhite192 In terms of file / folder naming, I think it makes sense to name the component files the same as the component name, so they'd both be capitalized. As for non-component files and folders, I think typically lower-case is most common but it's really up to you, either way consistency is the key so you can quickly hop around folders as you go!

**Comment by AccessiT3ch** [comment]
@mwhite192 One thing I would also say about the naming convention for components is that they should be as specific as they need to be. I would think `AppFooter` would just be `Footer` b/c it's so global, and then any other footer component you need to make would be more specifically named, like `ModalFooter` or `SettingsSideBarFooter`, etc.

I'm also not sure what `CredForms` are based on the name alone, if the component is responsible for both login and signup, it could be named `Auth` or something :)

**Comment by mwhite192** [comment]
@AccessiT3ch - Thank you so much for that Conor!  I changed the name of 'CredForms' to 'AuthForms'.  I will try to make sure I keep the naming of components specific, but not too crazy.  I also changed 'AppFooter' to 'Footer'.  As far as the file/ folder naming, it seems I got that down okay making me good to go there.  Thanks for the tip relating to non-component files.  That was something I didn't know or hear about.
